### PR TITLE
Fix issue with group's description

### PR DESCRIPTION
**Pull request by fabianrbz**
This also fixes an issue with a group's description

### CODE REVIEW

File: lib/nexmo/oas/renderer/views/open_api/_model.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <div class="Vlt-col Vlt-col--2of3 Nxd-api__docs">
     <div>
       <h2 id="<%= group['name'].parameterize %>" class='Vlt-grey-dark'><%= group['name'] %></h2>
-      <p class: "Vlt-grey-darker"><%= group['description'].render_markdown %></p>
+      <p class: "Vlt-grey-darker"><%= group['description']&.render_markdown %></p>
```
Comment (benhayehudi) [inline lib/nexmo/oas/renderer/views/open_api/_model.erb]: Is this because sometimes `group['description']` was/is Nil?

Comment (fabianrbz) [reply lib/nexmo/oas/renderer/views/open_api/_model.erb]: yes :D

Comment (fabianrbz) [reply lib/nexmo/oas/renderer/views/open_api/_model.erb]: @benhayehudi  if you run it locally with:

```
OAS_PATH=../api-specification/definitions bundle exec shotgun lib/nexmo/oas/renderer/config.ru
```

and visit http://127.0.0.1:9393/vonage-business-cloud/vgis

you'll get the error

Comment (benhayehudi) [reply lib/nexmo/oas/renderer/views/open_api/_model.erb]: Thanks, that's what I thought, just wanted to make sure 👍

**Pull request merged**